docs/documentation/logging.rst

```diff
@@ -0,0 +1,60 @@
+.. _logging:
+
+Logging
+=======
+
+Logging in audiomate is done using the standard python logging facilities.
+
+Enable Logging
+--------------
+
+By default, only messages of severity ``Warning`` or higher are printed to ``sys.stderr``.
+At some points audiomate provides more detailed info and debug information.
+In order to enable messages for levels of lowere severity, logging has to be configured.
```

Comment:
`To enable logging of messages of lower severity, configure Python's logging system as follows:`. Then example, `For further information…` after the code block. It's a bit easier to read.